cmd/bootstrap/PROVIDED_CERTIFICATES.md

```diff
@@ -0,0 +1,150 @@
+# User provided certificates
+
+Users can provide their own certificates to be used with Skupper V2 in non kube sites
+during the bootstrap of a local site, when preparing a site bundle and even while installing
+a site bundle at a remote machine. 
+
+## How certificates are used internally
+
+During the initialization of a local non kube site or when a site bundle is being prepared,
+Skupper generates an internal Certificate Authority (CA) named `skupper-site-ca`.
+
+If a `RouterAccess` is defined as part of the non kube site definition, Skupper will also
+generate a server certificate that is valid for the provided `RouterAccess.spec.bindHost` and
+for each entry in the  `RouterAccess.spec.subjectAlternativeName` list. The respective certificate
+is signed by the CA mentioned earlier (skupper-site-ca).
+
+A set of static links are also created for the provided `RouterAccess.spec.bindHost` and for each
+entry in the  `RouterAccess.spec.subjectAlternativeName` list as separate YAML files.
+
+These static links are composed by a `Link` and a `Secret`, which is basically the client certificate
+signed by `skupper-site-ca`, that will be used by other sites to establish Skupper links.
+
+After a site bundle has been produced, it contains the whole site definition that can be installed
+at a remote location. Thus, all certificates and static links are already part of the bundle and no
+new certificate is signed at the moment a bundle is installed.
+
+## Providing your own certificates
+
+You can provide your own certificates to be used by Skupper for site linking.
+Depending on your goals, some certificates should be supplied at certain phases,
+for example, at the time a local site is being initialized, a bundle is being prepared
+or a bundle is being installed at a remote location.
+
+To understand it better, let's go through the main use cases and review what is the ideal
+phase that a given kind of certificate should be provided.
+
+### Using a custom CA to sign certificates
+
+If you want Skupper to generate client and server certificates signed by a custom CA,
+you will need to provide the respective certificates during:
+
+* Local site initialization time
+* Site bundle preparation time
+
+Certificates are signed by Skupper at the time a local namespace is being initialized
+or a bundle is being produced.
+
+After a site bundle has been produced, it already contains the whole definition and no
+new certificate is expected to be signed.
+
+### Using custom server and client certificates
+
+If you want a new local site to use your custom server and client certificates, you can
+provide them at any time, for example:
+
+* Local site initialization time
+* Site bundle preparation time
+* Site bundle installation time
+
+During "Local site initialization time" or "Site bundle preparation time", Skupper will detect
```

Comment:
Again, maybe worth clarifying here *how* it detects this? Also if it detects a server certificate will the CA still be generated or is that then deemed unnecessary?

Reply:
I will update it, explaining how they are detected.

Just answering your question, the CA is always generated (if not provided).

The server certificate is currently associated with a RouterAccess, so users might eventually
provide server and client certificates to a given RouterAccess but they could have a different
RouterAccess defined that uses certificates generated by the default CA.